span-processing-config-service-api/src/main/proto/org/hypertrace/span/processing/config/service/v1/span_processing_config_service.proto

```diff
@@ -149,3 +150,46 @@ message ApiNamingRuleMetadata {
   google.protobuf.Timestamp creation_timestamp = 1;
   google.protobuf.Timestamp last_updated_timestamp = 2;
 }
+
+message SamplingConfig {
```

Comment:
One issue that I'm contemplating is - If there are conflicting sampling configs at the same hierarchy(say we define 2 conflicting sampling configs for the same environment), how do we handle it ?🤔
Since, this override mechanism can be done by traceable admins alone, should we assume this would not be the case ?